internal/services/containerapps/container_app_resource_test.go

```diff
@@ -1666,6 +1702,80 @@ resource "azurerm_container_app" "test" {
 `, r.template(data), data.RandomInteger)
 }
 
+func (r ContainerAppResource) ingressSecurityRestriction(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_container_app" "test" {
+  name                         = "acctest-capp-%[2]d"
+  resource_group_name          = azurerm_resource_group.test.name
+  container_app_environment_id = azurerm_container_app_environment.test.id
+  revision_mode                = "Single"
+
+  template {
+    container {
+      name   = "acctest-cont-%[2]d"
+      image  = "jackofallops/azure-containerapps-python-acctest:v0.0.1"
+      cpu    = 0.25
+      memory = "0.5Gi"
+    }
+  }
+
+  ingress {
+    target_port = 5000
+    ip_security_restriction = [
+      {
+        name             = "test"
+        description      = "test"
+        action           = "Allow"
+        ip_address_range = "0.0.0.0/0"
+      }
+    ]
```

Comment:
This needs to be specified as a block, and needs at least one `traffic_weight` block, so I think this would be correct:
```suggestion
    ip_security_restriction {
        name             = "test"
        description      = "test"
        action           = "Allow"
        ip_address_range = "0.0.0.0/0"
      }

    traffic_weight {
      latest_revision = true
      percentage      = 100
    }  
```
(Check the whitespace there, GH interface is not the best at getting these suggestions correct for the terraform linters)